### PR TITLE
Update copyright year 2023

### PR DESCRIPTION
**Pull request by Seb33300**
### Summary

Happy New Year 2023! 🎆

### TODO:
- [ ] CHANGELOG(s) updated with appropriate info
- [ ] Just before pressing the merge button, ensure new entries to CHANGELOG(s) are still under the _UNRELEASED_ heading

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -141,4 +141,4 @@ Holochain is licensed under the Cryptographic Autonomy License [![License: CAL v
 
 Other components, applications, and libraries we build are typically shared under the [Apache License v2](http://www.apache.org/licenses/LICENSE-2.0) as a simple, lighweight, and flexible way to share code.
 
-Copyright (C) 2017 - 2021, Holochain Foundation
+Copyright (C) 2019 - 2023, Holochain Foundation
```
Comment (Seb33300) [inline CONTRIBUTING.md:144]: Not sure about this one, this is the only one that was using `2017`.
I aligned it to 2019 like all others.

**Comment by abe-njama** [comment]
@Seb33300 would you like to merge this PR? We are cutting a release today and these updates seem necessary to go in.

**Comment by Seb33300** [comment]
Yes you can merge it

**Review by maackle — APPROVED** [review]
Thanks!

**Pull request merged**